packages/iris-grid/src/IrisGridTreeTableModel.ts

```diff
@@ -231,9 +240,10 @@ class IrisGridTreeTableModel extends IrisGridTableModelTemplate<
   }
 
   get layoutHints(): DhType.LayoutHints | null | undefined {
-    // TODO: Update this when JS API types are updated with this.
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    return (this.table as any).layoutHints;
+    if (isLayoutTreeTable(this.table)) {
+      return this.table.layoutHints;
+    }
+    return undefined;
```

Comment:
I realized it shouldn't really matter since DHE would just return undefined for `this.table.layoutHints`. But still not bad to feature detect it.

Reply:
Yea my thoughts as well